test/ttmlir/Conversion/StableHLOToTTIR/round_op.mlir

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+module @jit_eltwise_round attributes {} {
+  func.func public @test_round(%arg0: tensor<4xbf16>) -> tensor<4xbf16> {
+    %0 = stablehlo.round_nearest_afz %arg0 : tensor<4xbf16>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
```

Comment:
`[[C:.*]] =` means store everything up to `=` into variable `C`, but we never use `C` so the simplest way to check if op appears in output of `ttmlir-opt` is just `// CHECK: tensor.empty`. Same for the other checks.